packages/voting-contract/src/state.rs

```diff
@@ -267,6 +267,7 @@ pub const PROPOSAL_COUNT: Item<u64> = Item::new("proposal_count");
 
 // multiple-item map
 pub const BALLOTS: Map<(u64, &Addr), Ballot> = Map::new("votes");
+pub const BALLOTS_BY_VOTER: Map<(&Addr, u64), Ballot> = Map::new("votes_by_voter");
```

Comment:
Is there a storage option that would let us store all the ballots in a single "map" with two indexes? Like the `IndexedMap` from `storage-plus`?

Reply:
Good idea, I think it allows indexing by multiple keys.
I'll check that out.

Reply:
This is how we [do it in trusted-circle](https://github.com/confio/tgrade-contracts/blob/main/contracts/tgrade-trusted-circle/src/state.rs#L613-L614) as well. I think this is fine for this PR as they are only written once and never updated.

Using IndexedMap for this one is a nice improvement, but can be pulled out to another issue (either for 0.6 or 0.7), and trusted circle should also be updated then. Can you make such an issue and link it? Add it to 0.6 "patch net stectch" milestone for starters

Reply:
Done. https://github.com/confio/poe-contracts/issues/83

Reply:
@ethanfrey 
> I think this is fine for this PR as they are only written once and never updated.

...although they are updated.
https://github.com/confio/poe-contracts/pull/79/files#diff-4658a3ea273c6042143567e7913e908f5166041b5a186a97516f90f6223b9f78R140
Does that change your position here?